locale/it/translation.js

```diff
@@ -0,0 +1,415 @@
+export const it_translation = {
+    // Les balises HTML de base sont autorisées, comme <b><i>, etc. Toutes les données sont nettoyées https://developer.mozilla.org/fr/docs/Web/API/Element/innerHTML
+
+    // NOTA: Si une section n'a PAS besoin d'être traduite, laissez-la vide..
+
+    // General words
+    amount: 'Montant', //Amount
+    staking: 'Staking', //Staking
+    wallet: 'Portafoglio', //Wallet
+    display: 'Schermo', //Display
+    activity: 'Attività', //Activity
+    yes: 'Sì', //Yes
+    no: 'No', //No
+
+    // Nav Bar
+    navDashboard: 'Dashboard', //Dashboard
+    navStake: 'Stake', //Stake
+    navMasternode: 'Masternode', //Masternode
+    navGovernance: 'Governance', //Governance
+    navSettings: 'Impostazioni', //Settings
+
+    // Footer
+    footerBuiltWithPivxLabs: 'Costruito col 💜 da PIVX Labs ', //Built with 💜 by PIVX Labs
+
+    // Intro
+    loading: 'Caricamento', //Loading
+    loadingTitle: 'Il mio Wallet PIVX', //My PIVX Wallet is
+
+    // Onboarding Dashboard
+    dashboardTitle: 'Dashboard', //Dashboard
+    dCardOneTitle: 'Crea un ', //Create a
+    dCardOneSubTitle: 'Nuovo Wallet', //New Wallet
+    dCardOneDesc:
+        'Creare un nuovo Wallet PIVX, offre metodi di sicurezza & di backup più sicuri .', //Create a new PIVX wallet, offering the most secure backup & security methods.
+    dCardOneButton: 'Crea un Nuovo Wallet', //Create A New Wallet
+
+    dCardTwoTitle: 'Crea un nuovo ', //Create a new
+    dCardTwoSubTitle: 'Wallet Vanity', //Vanity Wallet
+    dCardTwoDesc:
+        'Creare un portafoglio con un prefisso personalizzato, ti permette di tenerlo più a lungo !', //Create a wallet with a custom prefix, this can take a long time!
+    dCardTwoButton: 'Crea un Wallet vanity', //Create A Vanity Wallet
+
+    dCardThreeTitle: 'Accedi al tuo ', //Access your
+    dCardThreeSubTitle: 'Portafoglio hardware', //Hardware Wallet
+    dCardThreeDesc:
+        "Utilizza il tuo portafoglio Ledger con un'interfaccia più familiare.", //Use your Ledger Hardware wallet with MPW's familiar interface.
+    dCardThreeButton: 'Accedi alla mia Ledger', //Access my Ledger
+
+    dCardFourTitle: 'Vai al ', //Go to
+    dCardFourSubTitle: 'Mio Wallet', //My Wallet
+    dCardFourDesc:
+        "Importa un portafoglio PIVX usando una Chiave Privata, xpriv, o una Seed Phrase  .", //Import a PIVX wallet using a Private Key, xpriv, or Seed Phrase.
+    dCardFourButtonI: 'Importa Portafoglio', //Import Wallet
+    dCardFourButtonA: 'Accedi al mio Portafoglio', //Access My Wallet
+
+    // Seed Phrase Creation
+    thisIsYourSeed: "Questa è la tua seed phrase:", //This is your seed phrase:
+    writeDownSeed:
+        'Scrivila da qualche parte. La vedrai solo <b>una volta!</b>', //Write it down somewhere. You'll only see this <b>once!</b>
+    doNotShareWarning:
+        "Chiunque ne ha una copia può accedere a <b> tutti </b> vostri fondi", //Anyone with a copy of it can access <b>all</b> of your funds.
```

Comment:
```suggestion
        "Chiunque ne ha una copia può accedere a <b> tutti </b> tuoi fondi", //Anyone with a copy of it can access <b>all</b> of your funds.
```